ci-operator/config/stolostron/search-v2-operator-bundle/stolostron-search-v2-operator-bundle-release-2.7.yaml

```diff
@@ -58,7 +58,7 @@ tests:
         export OSCI_IMAGE_REMOTE_REPO_SRC="registry.ci.openshift.org/stolostron"
         export SELF="make"
         export OSCI_PUBLISH_DELAY="0"
-        make -f Makefile.prow osci/publish
+        make -f /opt/build-harness/Makefile.prow osci/publish
```

Comment:
This should work, but for consistency, our other projects have this in Makefile.prow

https://github.com/stolostron/search-v2-api/blob/main/Makefile.prow#L3

Reply:
I also noticed that the project is missing `Makefile.prow` it only has `Makefile`

Reply:
Updated the reference to point to `Makefile.prow`.